index.js

```diff
@@ -18,18 +18,14 @@ const kebabCase = '([a-z][a-z0-9]*(-[a-z0-9]+)*)'; // kebab-case
 const camelCase = '([a-z0-9]+([A-Z][a-z0-9]+)*)'; // camelCase
 
 module.exports = {
-  extends: ['stylelint-config-standard', 'stylelint-prettier/recommended'],
+  extends: ['stylelint-config-standard-scss', 'stylelint-prettier/recommended'],
   plugins: [
-    '@skyscanner/stylelint-plugin-backpack',
```

Comment:
`@skyscanner/stylelint-plugin-backpack` was uninstalled. It was disabled and did not support `v14`, uninstall rather than wait on an update.

Reply:
Ah yeah this was something on the backburner and served constantly as a reminder it needed fixing!